node-notes-cli/app.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const data = require('./data.json');
+
+function err() {
+  if (err) {
+    return err;
+  }
+}
```

Comment:
if there is an error, it's a good idea to print it to the terminal for debugging purposes.